lib/git-log.js

```diff
@@ -38,7 +38,9 @@ const tags = require('./tags');
  * See https://git-scm.com/docs/pretty-formats
  */
 exports.GIT_LOG_YAML_PRETTY_FORMAT = [
-  '- subject: >-',
+  '- hash: >-',
```

Comment:
The hash is a single string, so we shouldn't be using YAML multi-string support here (the `>-`). Something like `- hash: %H` should suffice.